kw/platform/requests_utils.py

```diff
@@ -0,0 +1,60 @@
+import os
+
+import wrapt
+
+
+APP_NAME = os.getenv("APP_NAME")
+PACKAGE_VERSION = os.getenv("PACKAGE_VERSION", "1.0.0")
+ENVIRONMENT = os.getenv("ENVIRONMENT", "dev")
```

Comment:
I think the default values won't really be useful, ever, and there's some chance that people will leave these values in the User-Agent.

Reply:
I agree. I just play with them during testing, I plan to leave these as empty string `""`. If they're not provided they'll be basically skipped. I don't think enforcing multiple variables would be good solution.

Reply:
Well, I think requesting multiple variables is the only solution though. If people leave some as empty string, they'll have a broken user agent string which will fail server-side validation.

Reply:
By the way I just realized, `APP_ENVIRONMENT` is probably a better name for this variable.